build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+ROOT=$(dirname $(readlink -f "$0"))
+export GOPATH=${ROOT}/.build
+
+git submodule update --init --recursive
+
+echo "Building in .build/ subdir"
+echo "GOPATH=$GOPATH"
+
+CGO_CFLAGS="-I$GOPATH/src/github.com/harmony-one/bls/include -I$GOPATH/src/github.com/harmony-one/mcl/include -I/usr/local/opt/openssl/include"
```

Comment:
We have `scripts/setup_bls_build_flags.sh` for this.  Let's concentrate the logic there (per DRY).